PROVISIONING.md

```diff
@@ -42,36 +42,36 @@
 
 see [Inventory](INVENTORY.md)
 
-## Re-Provisioning
+## Adopted solution for RFC 8572 Secure Zero Touch
 
-tbd... re-celling, faults...
+See [ZTP](ZTP.md) for diagrams and detailed explanation.
 
-## Manual Provisioning
+See [Demo](https://github.com/opiproject/opi-prov-life/tree/main/examples/sztp) for running example.
 
-Andy: Do we need to cover manual methods in our scope? Many, many methods...
+## Additional Provisioning methods out of OPI scope
```

Comment:
Same problem, should be `Additional provisioning methods out of OPI scope`.

Reply:
fixed